tools/docker-compose/Dockerfile

```diff
@@ -2,59 +2,125 @@ FROM centos:7
 
 ARG UID=0
 
-ADD Makefile /tmp/Makefile
-
-RUN mkdir /tmp/requirements
-
 RUN yum -y update && yum -y install epel-release
-RUN yum -y localinstall https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-redhat96-9.6-3.noarch.rpm
 
 # sync with installer/roles/image_build/templates/Dockerfile.j2
-RUN yum -y install ansible git mercurial subversion curl python-psycopg2 python-pip python-setuptools libselinux-python setools-libs yum-utils sudo acl make postgresql-devel nginx python-psutil libxml2-devel libxslt-devel libstdc++.so.6 gcc cyrus-sasl-devel cyrus-sasl openldap-devel libffi-devel python-pip xmlsec1-devel swig krb5-devel xmlsec1-openssl xmlsec1 xmlsec1-openssl-devel libtool-ltdl-devel bubblewrap gcc-c++ python-devel python36-setuptools python36-devel krb5-workstation krb5-libs libcurl-devel rsync unzip
+RUN yum -y install acl \
```

Comment:
Instead of one per-line, could we just wrap this at 80 chars or something?  This is pretty tall.

Reply:
I personally prefer this approach because it keeps diffs cleaner and it makes it easier to keep the list sorted. When you format things to 80 chars now, it will grow into another unreadable, unsorted glob that no one ever really looks at.